dspace_rest_client/models.py

```diff
@@ -517,4 +517,72 @@ class RelationshipType(AddressableHALResource):
     def __init__(self, api_resource):
         super(RelationshipType, self).__init__(api_resource)
 
+class License(AddressableHALResource):
+    """
+    Specific attributes and functions for licenses
+    """
+    type = 'clarinlicense'
```

Comment:
TT

Reply:
I wanted to change it because I understand the reasons, but when I checked the entire file where I added it, each of the classes was made similarly. In this case, wouldn't it be better to do it as the author did?